app/src/main/res/layout-land/coming_soon_topic_view.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.home.promotedlist.ComingSoonTopicsViewModel" />
+  </data>
+
+  <com.google.android.material.card.MaterialCardView
+    android:id="@+id/topic_container"
+    android:layout_width="144dp"
+    android:layout_height="wrap_content"
+    app:layoutMarginEnd="@{viewModel.endPadding}"
```

Comment:
You are setting padding to marginEnd? It should be either padding or margin only.

Reply:
Done.